electron/src/lib/openGraph.ts

```diff
@@ -64,12 +64,15 @@ const fetchOpenGraphData = (url: string): Promise<OpenGraphResult> => {
   };
 
   return new Promise<string>((resolve, reject) => {
-    const getContentRequest = request.get(urlUtil.format(normalizedUrl), (error, response, body: string) => {
-      return error ? reject(error) : resolve(body);
```

Comment:
This was basically always resolving since we had `.on('data')` with a `resolve` and this callback.